content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/tabs.html

```diff
@@ -18,17 +18,21 @@
      class="cmp-tabs"
      data-cmp-is="tabs">
     <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
+        data-sly-list.tab="${tabs.items}"
         role="tablist"
         class="cmp-tabs__tablist"
+        aria-label="${tabs.label}"
         aria-multiselectable="false">
-        <li data-sly-repeat.item="${tabs.items}"
-            role="tab"
-            class="cmp-tabs__tab${item.name == tabs.activeItem ? ' cmp-tabs__tab--active' : ''}"
-            data-cmp-hook-tabs="tab">${item.title}</li>
+        <sly data-sly-test.isActive="${tab.name == tabs.activeItem}"/>
+        <li role="tab"
+            class="cmp-tabs__tab${isActive ? ' cmp-tabs__tab--active' : ''}"
+            tabindex="${isActive ? '0' : '-1'}"
+            data-cmp-hook-tabs="tab">${tab.title}</li>
     </ol>
     <div data-sly-repeat.item="${tabs.items}"
          data-sly-resource="${item.name @ decorationTagName='div'}"
          role="tabpanel"
+         tabindex="0"
```

Comment:
This was already added in https://github.com/adobe/aem-core-wcm-components/commit/29846f754886d6d0ed08eac3e2d080de590bd8dd

Reply:
I'm not sure either, this branch was made using only the development branch

Reply:
Weird. And if I do a local diff against development, I don't see these changes.